### DISCUSSION TITLE
Help with Topics and official website cache reset

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5068] [Help and Questions] [answered]
How much time does the official website need to refresh its cache about Scoop buckets??

I've just set the topics for my bucket and after a few hours looked up at https://scoop.sh/#/buckets for my bucket and it is not there. I did the same thing with app search on https://scoop.sh/#/apps?s=0&d=0&o=false&q=pb2 and the result is the same.

![b](https://user-images.githubusercontent.com/22171451/181045841-2eb0625c-0506-4b15-8678-27dbd1c70cce.PNG)

Does the webpage refresh daily at midnight or something like that?

**Comment by gpailler** [comment, marked as answer]
@anderlli0053 The index is updated 4 times per day so a new bucket should appear after 6 hours in the worst case.
Could you post a link to your bucket so I can check if there is any issue with the indexing?

Thanks

**Reply by anderlli0053** [reply]
> @anderlli0053 The index is updated 4 times per day so a new bucket should appear after 6 hours in the worst case. Could you post a link to your bucket so I can check if there is any issue with the indexing?
> 
> Thanks


Here it is:

https://github.com/anderlli0053/DEV-tools

**Reply by gpailler** [reply]
I checked quickly and I think there is some indexing lag on GitHub side.

I can find your repo with the following query: https://api.github.com/search/repositories?q=scoop+user%3Aanderlli0053+created%3A%3E2022-06-27 but not with this one (similar to the one used by our indexer): https://api.github.com/search/repositories?q=scoop+created%3A%3E2022-06-27

So let's wait until tomorrow to see if your repo appears when querying the GitHub API.

**Reply by anderlli0053** [reply]
I totally agree, i will report back tomorrow and we'll see if issues persists,

Thanks for the help,

Andrew

**Reply by anderlli0053** [reply]
[SOLVED]

It is solved, it took that much time to properly register it among other buckets. I assume that it was the GIT caching error / problem.

Thank you very much @gpailler  and @rashil2000 .
